static/phyui/manual_clustering.js

```diff
@@ -104,10 +104,25 @@ define(function (require) {
 
     events.on('kernel_connected.Kernel', function(){
         console.log('session done');
-        myhack.create_result_cell('#placeholder1', "from phyui.session import start_manual_clustering; session = start_manual_clustering('/home/ctaf/src/cortex/data/test_hybrid_120sec.kwik', backend='ipynb_webgl')");
-        myhack.create_result_cell('#placeholder2', "import vispy; \
-                                                    vispy.app.use_app('ipynb_webgl'); \
-                                                    w = session.show_waveforms()");
+        myhack.create_result_cell('#placeholder1', "from phyui.session import start_manual_clustering; \
+                                                    session = start_manual_clustering('/home/ctaf/src/cortex/data/test_hybrid_120sec.kwik', backend='ipynb_webgl')");
+        var cc = myhack.create_result_cell('#placeholder2', "import vispy; \
+                                                             vispy.app.use_app('ipynb_webgl'); \
+                                                             w = session.show_waveforms()");
```

Comment:
hardcoded path: I'am going to change that.

mixing python and js mean code reuse in that case. same widget used in the same way. that how you instanciate widget in the notebook, so great flexibility. that's what happen in the notebook. 
The idea being to be able to reuse widget from IPython a lot, maybe after the split it will make more sence to do otherwize, but still we will depends on a python kernel running our code.


using comm seems more complicated do you see an advantage to use it?
to me it seems lower level, so harder to maintain.